README.md

```diff
@@ -121,7 +122,7 @@ cd Palisade-common
 
 You are then ready to build with Maven:
 ```bash
-mvn install
+mvn clean install
```

Comment:
As this is likely to be a fresh clone of the Common repo the clean phase is not really necessary.

Reply:
The instructions are for a first time, but I think this is just a good habit to always include the clean.  It sets it in the commands that can be recalled when you inevitably change something and want to re-build.

Reply:
I am also happy to keep the clean in here